modules/build/src/main/scala/scala/build/input/Inputs.scala

```diff
@@ -304,17 +312,27 @@ object Inputs {
         else
           Left(s"""$arg does not contain shebang header
                   |possible fixes:
-                  |  Add '#!/usr/bin/env scala-cli shebang' to the top of the file
+                  |  Add '#!/usr/bin/env $fullProgramCall' to the top of the file
```

Comment:
This error shouldn't be displayed on Windows.  

`#!/usr/bin/env $fullProgramCall` it doesn't work on Windows.

Reply:
But still, even on Windows we will check whether the first line is a shebang header when `scala-cli shebang` is used. Should we suggest that the Windows users put just "#!" in the first line?